stock_quant_manual_assign/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Mikel Arregi - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Stock - Manual assignment of quants",
+    "version": "8.0.1.0.0",
```

Comment:
OK, understood. Then it's correct, except you can omit the line in the README.md, that is going to be changed in the next automated cron that scans modules.